src/MassTransit/DependencyInjection/Configuration/RiderRegistrationContext.cs

```diff
@@ -2,24 +2,22 @@ namespace MassTransit.Configuration
 {
     using System;
     using System.Collections.Generic;
-    using System.Linq;
+    using Microsoft.Extensions.DependencyInjection;
 
 
     public class RiderRegistrationContext :
         IRiderRegistrationContext
     {
         readonly IRegistrationContext _registration;
-        readonly IRegistrationCache<object> _registrations;
```

Comment:
@NooNameR I'm thinking this change actually broke registration, to where rider-registered consumers are showing up in the bus and being registered as receive endpoints via ConfigureEndpoints. The registrations seem to be co-mingled.